site/content/api/Command-And-Control-API.md

```diff
@@ -40,33 +39,24 @@ The following table provides an overview of the properties the *Business Applica
 | *subject* | yes | *properties* | *string* | MUST contain the command to be executed by a device. |
-| *content-type* | no | *properties* | *symbol*  | SHOULD be set to *application/octet-stream* if the message payload is to be considered *opaque* binary data. In most cases though, the client should be able to set a more specific content type indicating the type and characteristics of the data contained in the payload, e.g. `text/plain; charset="utf-8"` for a text message or `application/json` etc. |
-| *ttl* | no | *properties* | *int* | Indicates the timeout value (in milliseconds) indicating when server can assume that command has not been executed because of the lack of the response from the client. |
+| *content-type* | no | *properties* | *symbol*  | SHOULD be set to *application/octet-stream* if the message payload is to be considered *opaque* binary data. In most cases though, the sender should be able to set a more specific content type indicating the type and characteristics of the data contained in the payload, e.g. `text/plain; charset="utf-8"` for a text message or `application/json` etc. |
+| *ttl* | no | *properties* | *int* | Duration in milliseconds for which the message is to be considered 'live'. Messages that live longer than their expiration time will be discarded (or dead lettered). |
 
 If a control message requires additional data to be properly executed (like command parameters), the body of the message MUST consist of a single AMQP *Data* section containing a control 
-message data. The format and encoding of the data is indicated by the *subject*. It is up to the device to interpret the incoming body by taking it into 
+message data. The format and encoding of the data is indicated by the *subject* and *content-type*. It is up to the device to interpret the incoming body by taking it into 
 consideration.
 
-## Sending back control message execution result
-
-After execution of a control message on a device, device MUST send a response message indicating a status of that execution.
-
-**Preconditions**
-
-1. *Protocol Adapter* and *Business Application* have established an AMQP connection with the AMQP 1.0 Network.
-2. *Protocol Adatper* has established an AMQP link in role *sender* with Hono using target address `control-reply/${tenant_id}/{$device_id}`. Where `${device_id}` is an ID of the
-device that received the control message and `${tenant_id}` is the ID of the tenant the device belongs to.
-3. *Business Application* has established an AMQP link in role *receiver* with the AMQP 1.0 Network. 
+## Receiving the Commands execution result
 
-(The message flow is shown in the sequence diagram above to show the whole sequence in one place)
+After execution of a control message on a device, the device MUST send a response message indicating a status of that execution.
```

Comment:
is there a difference between a *command* and a *control* message? If so, IMHO we need to define these terms at the top of the document. It is quite confusing to read about *command* in one place and *control message* in the other without knowing whether these terms are synonyms to each other not.

Reply:
There is no difference. As it was in the existing API description I understood it as sending a command which includes a control message. But I am open to always speak of a command.

Reply:
I think that would be a good idea and make the whole document more consistent.

FMPOV we should also explain the difference between commands to which the application expects a response vs. *one-way* commands and if and how an application knows whether the command has been delivered to the device.

Reply:
I decided to just start with commands, which always expects a response.

Reply:
The response message, send by the device which at least includes the status code.